src/test/javascript/spec/component/exam/manage/student-exams/student-exam-detail-table-row.component.spec.ts

```diff
@@ -0,0 +1,129 @@
+import { StudentExamDetailComponent } from 'app/exam/manage/student-exams/student-exam-detail.component';
```

Comment:
`'StudentExamDetailComponent' is defined but never used`

Reply:
Done